index.html

```diff
@@ -3,34 +3,51 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio Website: Ademola Adebayo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Portfolio: Ademola Adebayo</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="new.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
 </head>
 <body>
-  <header class="main-head">
-    <nav class="toolbar">
-      <div class="upper-toolbar">
-        <div><h2>Smart City</h2></div>
-        <div class="nav-links">
-          <ul>
-            <li><a href=""></a>Portfolio</li>
-            <li><a href=""></a>About</li>
-            <li><a href=""></a>Contact</li>
-          </ul>
-          <a href=""><i class=" far fa-envelope fa-1em"></i></a> 
-        </div>
+  <header class="main">
+
+    <nav class="mobile">
+
+      <img src="./close.svg" alt="" class="close">
+      
+      <div class="menu-right">
+        <ul class="menu-list">
+          <li><a class="menu-links" href="#portfolio">Portfolio <img src="./right.svg" alt=""></a></li>
+          <li><a class="menu-links" href="#about">About <img src="./right.svg" alt=""></a></li>
+          <li><a class="menu-links" href="#contact">Contact <img src="./right.svg" alt=""></a></li>
+        </ul>
       </div>
-      <div class="top-toolbar">
-        <div><h2>Smart City</h2></div>
-        <a href=""><i class=" fas fa-bars fa-2x"></i></a>
+    </nav>
+
+    <nav class="navbar">
+      <div class="nav-left">
+       
+          <h2>Smart City</h2>
+        
       </div>
-    </nav>  
+      <div class="nav-right">
+        <ul class="nav-links">
+          <li><a class="links" href="#portfolio">Portfolio</a></li>
+          <li><a class="links" href="#about">About</a></li>
+          <li><a class="links" href="#contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] You have done a great job concerning the mobile menu, however, in the process, the desktop navbar seems to have gotten a little bit distorted. Kindly justify the nav contents properly.
Here is the :
### FIGMA Template
![Screenshot from 2022-04-13 17-52-55](https://user-images.githubusercontent.com/58692590/163232457-7f8a0ed1-a205-46c3-9e10-47698b947023.png)
### Your Implementation
![Screenshot from 2022-04-13 17-53-02](https://user-images.githubusercontent.com/58692590/163232598-930c9c43-e674-4081-be8a-0fb74ac31162.png)